[0.000000] (-) TimerEvent: {}
[0.003164] (tcp_server_xml) JobQueued: {'identifier': 'tcp_server_xml', 'dependencies': OrderedDict()}
[0.003460] (tcp_server_xml) JobStarted: {'identifier': 'tcp_server_xml'}
[0.099126] (-) TimerEvent: {}
[0.199657] (-) TimerEvent: {}
[0.300224] (-) TimerEvent: {}
[0.400783] (-) TimerEvent: {}
[0.501373] (-) TimerEvent: {}
[0.601931] (-) TimerEvent: {}
[0.702533] (-) TimerEvent: {}
[0.803098] (-) TimerEvent: {}
[0.903611] (-) TimerEvent: {}
[1.004112] (-) TimerEvent: {}
[1.104632] (-) TimerEvent: {}
[1.119481] (tcp_server_xml) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 67, in execute\n    loop.run_until_complete(future)\n  File "/usr/lib/python3.10/asyncio/base_events.py", line 633, in run_until_complete\n    self.run_forever()\n  File "/usr/lib/python3.10/asyncio/base_events.py", line 600, in run_forever\n    self._run_once()\n  File "/usr/lib/python3.10/asyncio/base_events.py", line 1860, in _run_once\n    event_list = self._selector.select(timeout)\n  File "/usr/lib/python3.10/selectors.py", line 469, in select\n    fd_event_list = self._selector.poll(timeout, max_ev)\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 105, in build\n    return await extension.build(additional_hooks=additional_hooks)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 81, in build\n    available_commands = await self._get_available_commands(\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 171, in _get_available_commands\n    output = await check_output(\n  File "/usr/lib/python3/dist-packages/colcon_core/subprocess.py", line 128, in check_output\n    assert not rc, f\'Expected {args} to pass: {stderr_data}\'\nAssertionError: Expected [\'/usr/bin/python3\', \'setup.py\', \'--help-commands\'] to pass: Traceback (most recent call last):\n  File "/home/klyx/git/p4/software_and_automation_frameworks_p4/mini_ws/src/tcp_server_xml/setup.py", line 1, in <module>\n    from setuptools import setup\n  File "/home/klyx/.local/lib/python3.10/site-packages/setuptools/__init__.py", line 16, in <module>\n    import setuptools.version\n  File "/home/klyx/.local/lib/python3.10/site-packages/setuptools/version.py", line 1, in <module>\n    import pkg_resources\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3267, in <module>\n    def _initialize_master_working_set():\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3241, in _call_aside\n    f(*args, **kwargs)\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3302, in _initialize_master_working_set\n    list(map(working_set.add_entry, sys.path))\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 613, in add_entry\n    for dist in find_distributions(entry, True):\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2077, in find_on_path\n    for dist in factory(fullpath):\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2142, in distributions_from_metadata\n    yield Distribution.from_location(\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2606, in from_location\n    )._reload_version()\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3004, in _reload_version\n    md_version = self._get_version()\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2785, in _get_version\n    version = _version_from_file(lines)\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2569, in _version_from_file\n    line = next(iter(version_lines), \'\')\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2780, in _get_metadata\n    for line in self.get_metadata_lines(name):\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 1422, in get_metadata_lines\n    return yield_lines(self.get_metadata(name))\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 1412, in get_metadata\n    value = self._get(path)\n  File "/home/klyx/.local/lib/python3.10/site-packages/pkg_resources/__init__.py", line 1616, in _get\n    with open(path, \'rb\') as stream:\nKeyboardInterrupt\n\n'}
[1.120266] (tcp_server_xml) JobEnded: {'identifier': 'tcp_server_xml', 'rc': 'SIGINT'}
[1.130576] (-) EventReactorShutdown: {}
